.gitignore

```
@@ -7,3 +7,5 @@ build/
 __pycache__/
 *.py[cod]
 .idea
```

Not strictly needed for this PR but it would be nice to include this in all repos.
```suggestion
.idea
venv/
```